AjobK/Athena
# Added MobX & Header closes #30

## Changes
- Added MobX
- Added Header

### How to test
- Open branch `30-header`
  - Run command `npm ci`
  - Run command `npm run dev`
  - Check the page & files for errors
